tools/templates/extras/xloop.jinja2

```diff
@@ -1,33 +0,0 @@
-def xloop(
```

Comment:
It looks like `xloop` was made private at some point and then removed. Let's let @jbachurski or @cbourjau weigh in briefly for some historical context first, but to my eye this looks fine.

Reply:
I don’t remember this very well, so it may have been an attempt to make the API more convenient. It’s probably fine to remove since it’s unused.